gui.lua

```diff
@@ -596,6 +596,9 @@ local function addCityOverview(city, anchor)
     -- citizen count
     tbl.add{type = "label", caption = {"", {"tycoon-gui-citizens"}, ": "}}
     tbl.add{type = "label", caption = {"", countCitizens(city)}}
+    -- debt
+    tbl.add{type = "label", caption = {"", {"tycoon-gui-debt"}, ": "}}
+    tbl.add{type = "label", caption = {"", string.format("%.2f", city.stats.debt)}}
```

Comment:
```suggestion
    tbl.add{type = "label", caption = {"", string.format("%.2f", city.stats.debt)}, tooltip = {"", {"tycoon-gui-debt-tooltip"}}}
```

Reply:
this `%.2f` format was put here to check that `x.4 + x.5 + x.6` correctly payouts as `1.0` and player never loses fractional coins.

it's simple to change to `%.0f`/`%d` or just `math.floor(city.stats.debt)`. i suppose we don't need `tostring(number)` here, but i'd use it (just in case it's `nil`):
```diff
-    tbl.add{type = "label", caption = {"", string.format("%.2f", city.stats.debt)}}
+    tbl.add{type = "label", caption = {"", tostring(math.floor(city.stats.debt))}}
```